ecs-agent/daemonimages/csidriver/driver/node.go

```diff
@@ -175,16 +191,21 @@ func (d *nodeService) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 	// Otherwise we need to create the target directory.
 	if !exists {
 		// If target path does not exist we need to create the directory where volume will be staged
-		klog.InfoS("NodeStageVolume: creating target dir", "target", target)
+		makedirtime := time.Now()
```

Comment:
related to my overall comment above, why is this useful?

Reply:
@singholt - Windows takes a bit more time to provision the volumes and times out for a few scenarios. We wanted to profile the code a bit for the first few weeks. We plan to remove these or convert them to debug statements after a few releases.

Reply:
sounds good.